underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -131,6 +131,12 @@ public void connectFromWorker(Configuration conf, String hostname) {
   public OutputStream create(String path) throws IOException {
     LOG.debug("Create method: {}", path);
     String newPath = path.substring(Constants.HEADER_SWIFT.length());
+    if (newPath.endsWith("_SUCCESS")) {
```

Comment:
@ifcharming We don't handle it different, the next line is empty object creation. I need to create an empty object, when _SUCCESS is created. Otherwise Spark will not be able to read those objects directly from Swift